doc/R.nvim.txt

```diff
@@ -1937,7 +1966,7 @@ Use `data.table` to save a TSV file:
 >lua
    view_df = {
      n_lines = 0,
-     save_fun = "function(obj, obj_name) {f <- paste0(obj_name, '.tsv'); data.table::fwrite(obj, f, sep = '\t') ; f}",
+     save_fun = "function(obj, obj_name) {f <- paste0('/tmp/', obj_name, '.tsv'); data.table::fwrite(obj, f, sep = '\t') ; f}",
```

Comment:
Is this just to fix the documentation?

Reply:
Yes, this is unrelated to the issue being fixed. It's a minor change that I was waiting to commit because I don't like to commit minor changes in the documentation. I prefer to accumulate some of them.